src/tests/commonTests.ts

```diff
@@ -338,16 +336,35 @@ export async function labelable(componentTagOrHtml: TagOrHTML, options?: Labelab
     <calcite-label for="${id}">${labelTitle}</calcite-label>
     ${componentHtml}
   `;
+  const siblingPage: E2EPage = await newE2EPage({ html: siblingHtml });
+  await siblingPage.waitForChanges();
 
   await assertLabelable({
-    html: siblingHtml,
+    page: siblingPage,
     componentTag,
     propertyToToggle,
     focusTargetSelector,
     shadowFocusTargetSelector
   });
-}
 
+  const componentFirstPage: E2EPage = await newE2EPage({ html: componentHtml });
+  await componentFirstPage.waitForChanges();
+  await componentFirstPage.evaluate((id: string) => {
+    const label = document.createElement("calcite-label");
+    label.setAttribute("for", `${id}`);
+    document.body.appendChild(label);
+  }, id);
+  const label = await componentFirstPage.find("calcite-label");
+  expect(await label.getProperty("for")).toBe(`${id}`);
+
+  await assertLabelable({
+    page: componentFirstPage,
+    componentTag,
+    propertyToToggle,
+    focusTargetSelector,
+    shadowFocusTargetSelector
+  });
```

Comment:
Do we need a test for the label-first case? 

How about the component-first, label-first cases for wrapped labelables?

Reply:
After our last discussion, the wrapped component-first (WCF) scenario might not be common, but the wrapped label-first one is. Can you add coverage for this case and add a comment explaining why WCF is not covered? 

Bonus points if you do cover WCF for completeness. :sparkles:⭐:sparkles: